ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -78,6 +101,244 @@ tests:
+        done
+
+        if [[ $(oc get csv -n openshift-cnv ${CSV} -o jsonpath={.status.phase}) != "Succeeded" ]]; then
+          echo "Error: Failed to deploy CNV"
+          echo "CSV ${CSV} YAML"
+          oc get CSV ${CSV} -n openshift-cnv -o yaml
+          echo
+          echo "CSV ${CSV} Describe"
+          oc describe CSV ${CSV} -n openshift-cnv
+          exit 1
+        fi
+
+        oc create -f - <<EOF
+        apiVersion: hco.kubevirt.io/v1beta1
+        kind: HyperConverged
+        metadata:
+          name: kubevirt-hyperconverged
+          namespace: openshift-cnv
+        EOF
+
+        oc wait hyperconverged -n openshift-cnv kubevirt-hyperconverged --for=condition=Available --timeout=15m
+      from: capk-oc-bin-image
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    - as: deploy-capk
+      commands: |-
+        set -x
+        # Deploy cluster API
+        cat <<EOF > clusterctl_config.yaml
+        cert-manager:
+          url: "https://github.com/cert-manager/cert-manager/releases/latest/cert-manager.yaml"
+        EOF
+
+        clusterctl init -v 4 --config=clusterctl_config.yaml
+
+        oc adm policy add-scc-to-user privileged system:serviceaccount:capk-system:capk-manager
+
+        export MANIFEST_IMG=${CAPK_IMAGE%:*}
+        export MANIFEST_TAG=${CAPK_IMAGE/*:}
+        GOFLAGS="-mod=readonly" make generate-manifests
+
+        sed -i -E "s|(^ +- image: ).*$|\1${CAPK_IMAGE}|" config/default/manager_image_patch.yaml
+
+        oc kustomize config/default | oc apply -f -
+        sleep 60
+
+        # Validate capk manager was created successfully
+        oc wait pod --for=condition=Ready -l cluster.x-k8s.io/provider=infrastructure-kubevirt -n capk-system --timeout=120s
+      dependencies:
+      - env: CAPK_IMAGE
+        name: cluster-api-kubevirt-controller
+      from: capk-oc-bin-image
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    - as: create-cluster
+      commands: |-
+        set -x
+        sed -i -E "/controlPlaneServiceTemplate:/{ n; /spec:/ { n; /type:/ { s/ClusterIP/LoadBalancer/; p; d;}}}" templates/cluster-template.yaml
+
+        export NODE_VM_IMAGE_TEMPLATE=quay.io/kubevirtci/fedora-kubeadm:35
+        export IMAGE_REPO=k8s.gcr.io
+        export CRI_PATH="/var/run/crio/crio.sock"
+        export CLUSTER_NAME=tenant-capk
+        clusterctl generate cluster ${CLUSTER_NAME} \
+                   --kubernetes-version v1.21.0 \
+                   --control-plane-machine-count=1 \
+                   --worker-machine-count=1 \
+                   --from templates/cluster-template.yaml \
+                   -n capk-system | oc apply -f -
+      from: capk-oc-bin-image
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    - as: validate-cluster
+      commands: |-
+        set -x
+        export CLUSTER_NAME=tenant-capk
+        export CLUSTER_NS=capk-system
+
+        sleep 60
```

Comment:
Why need sleep here, you have wait next line

Reply:
Because I want to make sure that resource exist before waiting for its conditions. The wait will fail if the resource does not exist.